### PR TITLE
Set Fan Mode

### PR DESCRIPTION
**Pull request by SShah2601**
Set ventilator mode to auto, minontime,  or on. (Disabled off
functionality)

### CODE REVIEW

File: pyecobee/__init__.py [hunk 1]
```diff
@@ -184,6 +184,26 @@ def set_fan_min_on_time(self, index, fan_min_on_time):
                   " fan minimum on time.  Refreshing tokens...")
             self.refresh_tokens()
 
+    def set_fan_mode(self, index, fan_mode):
+        ''' The minimum time, in minutes, to run the fan each hour. Values: auto, minontime, on, off '''
```
Comment (nkgilley) [inline pyecobee/__init__.py]: Update description

File: pyecobee/__init__.py [hunk 2]
```diff
@@ -184,6 +184,26 @@ def set_fan_min_on_time(self, index, fan_min_on_time):
...
+        params = {'format': 'json'}
+        body = ('{"selection":{"selectionType":"thermostats","selectionMatch":'
+                '"' + self.thermostats[index]['identifier'] +
+                '"},"thermostat":{"settings":{"vent":"' + fan_mode +
+                '"}}}')
+        request = requests.post(url, headers=header, params=params, data=body)
+        if request.status_code == requests.codes.ok:
+            return request
+        else:
+            print("Error connecting to Ecobee while attempting to set"
```
Comment (nkgilley) [inline pyecobee/__init__.py]: Update

**Comment by nkgilley** [comment]
Sorry those comments should just be for lines 188 and 203.

**Comment by SShah2601** [comment]
No worries. I updated the error message and description.

**Pull request merged**